scripts/install-helpers/baremetal-coco/osc_catalog.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: osc-upstream-catalog
+  namespace: openshift-marketplace
+spec:
+  displayName: OSC Upstream Operator Catalog
+  sourceType: grpc
+  image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-catalog:1.8.1-1
```

Current (and probably GA) build is 1.8.1-3.
```suggestion
  image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-catalog:1.8.1-3
```